src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1149,6 +1153,24 @@ private String execDockerCmd(String command, int timeout) {
         return result;
     }
 
+    // Suggest a performance improvement if docker build takes too long.
+    static final long DOCKER_BUILD_SOFT_TIMEOUT = 30000;
+    private void dockerIgnoreHelpMessage(long startTime) {
+        if (System.currentTimeMillis() - startTime < DOCKER_BUILD_SOFT_TIMEOUT) {
+            return;
+        }
+        File dockerfileToUse = dockerfile != null ? dockerfile : defaultDockerfile;
+        if (dockerfileToUse.exists()) {
+            File dockerContext = dockerfileToUse.getParentFile();
+            debug("dockerIgnoreHelpMessage, dockerContext="+dockerContext.getAbsolutePath());
+            File dockerIgnore = new File(dockerContext, ".dockerignore");
+            if (!dockerIgnore.exists()) { // provide some advice
+                warn("The docker build command is slower than expected. You may increase performance by adding " + 
+                    "unneeded files and directories such as any Liberty runtime directories to the .dockerignore file.");
```

Comment:
Should this mention what directory the .dockerignore file should go in? e.g. include the `dockerContext` path in the message.

Reply:
Yes, that would be helpful.